tests/exp_test.py

```diff
@@ -19,7 +19,7 @@ def test_exp_func_no_delim():
 
 
 def test_exp_command():
-    assert_equal("\\exponentialE(3)", exp(3))
```

Comment:
@builtbywill so would \\exponentialE(3) ever be allowed or is that the bug that we should not have allowed it?

Reply:
@lily514 I think technically it would still work but would be treated like e*3

Reply:
But yeah, that is the issue, that we were treating it as a func name instead of the symbol

Reply:
scratch that about it working 😂  no it will not be valid now

Reply:
double scratch that, i was right the first time (apparently I can't understand how unit tests work right now)

Reply:
I was just thinking it might be valuable to have some more unit tests here modeling what is/is not allowed.

Reply:
@lily514 sure! I can add back the original one with what it actually parses as

Reply:
@lily514 test cases added/updated!